addons/stock/tests/test_move.py

```diff
@@ -3576,7 +3401,7 @@ def test_edit_reserved_move_line_4(self):
         """ Test that editing a stock move line linked to an owned product correctly and directly
         adapts the reservation. In this case, we edit the owner to another available one.
         """
-        owner1 = self.env['res.partner'].create({'name': 'test_edit_reserved_move_line_4_1'})
+        owner1 = self.partner_1
         owner2 = self.env['res.partner'].create({'name': 'test_edit_reserved_move_line_4_2'})
```

Comment:
Since it's done a few times, maybe it would make sense to have a second partner in the `setUpClass`?

Reply:
Added `partner_2` to `setUpClass` and used both directly without `owner1,owner2` :+1: